### DISCUSSION TITLE
UX: list-keys option and less manual key(files) management

### DISCUSSION THREAD

**Discussion by alxiong** [FiloSottile/age#247] [UX reports]
1. similar to `gpg --list-keys`, would be nice if there's a canonical path to all `age`'s keys (e.g. `~/.config/age/keys.txt`) and default delimiter would be added between keys when invoking `age-keygen` so that a simple parsing and regex matching could support `list-key` command

2. in general, key(file) management still involves a bit of manual work, which could be deliberately by design (idk) for simplicity or maybe other reasons, but as a user, it will be great to not care about details like #75 .
(I guess I was used to the `ssh-keygen` way)

**Comment by FiloSottile** [comment]
The absence of a keyring is intentional. Keys are cheap to make, so it's expected users will make new key pairs for each application/use, and explicitly refer to them as files. Avoiding GPG's implicit keyring semantics was an original goal of the design.

**Reply by alxiong** [reply]
I see, understood. Your reply here https://github.com/FiloSottile/age/pull/104#issuecomment-825428387 is pretty clear. 
Thanks!
